utils/bootstrap.js

```diff
@@ -4,15 +4,26 @@ const pluginsService = require("../api/services/plugins.service");
 const socketService = require("../api/services/socket.service");
 const register = require("../utils/register");
 
-module.exports = () => {
+module.exports = async () => {
   logger.info("Loading plugins modules...");
-  pluginsService.loadAllInsalledPlugins().catch(console.error);
+  try {
+    await pluginsService.loadAllInsalledPlugins();
+  } catch (error) {
+    console.error(error)
+  }
   logger.info("Finish loading plugins");
   logger.info("Sending key to server");
-  register
-    .registerAgent()
-    .then(() => {
+  if(process.env.NODE_ENV !== "test") {
```

Comment:
I don't think we should test by using selectable variables from the actual execution perspective. Those call should be mocked in tests instead.